src/inventory_ui.cpp

```diff
@@ -3098,7 +3098,7 @@ item_location inventory_pick_selector::execute()
             if( highlight( input.entry->any_item() ) ) {
                 ui_manager::redraw();
             }
-            if( input.action == "SELECT" ) {
+            if( input.action == "SELECT" || input.action == "ANY_INPUT" ) {
```

Comment:
This still doesn't work for invlets that shadow other actions, for example `q` (for quit) and `r` (for reset filter).

`if( input.action != "MOUSE_MOVE" && input.action != "COORDINATE" )` should restore the old behaviour in this case as well.